pkg/manifests/manifests.go

```diff
@@ -24,15 +25,42 @@ import (
 	"github.com/Azure/aks-app-routing-operator/pkg/util"
 )
 
-const IngressClass = "webapprouting.kubernetes.azure.com"
+const (
+	IngressClass                = "webapprouting.kubernetes.azure.com"
+	deploymentKind              = "Deployment"
```

Comment:
nit: I personally find it more expressive to keep the kinds in struct string literals since they're commonly seen as literals in yaml manifests. Wouldn't block on it so do what you will. :)

Dig the other consts though - no more digging through the manifest to find the image tag.

Reply:
we're changing this a good deal with @OliverMKing 's next PR anyway, but I've personally run into issues when my string for the kind of resource wasn't exactly what it needs to be, so I wanted to keep a few of them as consts.